src/multibody/actions/free-fwddyn.cpp

```diff
@@ -90,12 +90,8 @@ void DifferentialActionModelFreeFwdDynamics::calcDiff(const boost::shared_ptr<Di
   const Eigen::VectorBlock<const Eigen::Ref<const Eigen::VectorXd>, Eigen::Dynamic> v = x.tail(nv);
 
   DifferentialActionDataFreeFwdDynamics* d = static_cast<DifferentialActionDataFreeFwdDynamics*>(data.get());
-  if (recalc) {
-    calc(data, x, u);
-    pinocchio::computeJointJacobians(pinocchio_, d->pinocchio, q);
```

Comment:
I am not sure if removing `computeJoinJacobians` have an implication. It seems the frame costs require this computation for getting the frame Jacobians.

Did you check the log files?

Reply:
Yes I did.